OrbitLinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -31,7 +31,7 @@ const std::array<size_t, unwindstack::X86_64_REG_LAST>
 std::vector<unwindstack::FrameData> LibunwindstackUnwinder::Unwind(
     unwindstack::Maps* maps,
     const std::array<uint64_t, PERF_REG_X86_64_MAX>& perf_regs,
-    const char* stack_dump, uint64_t stack_dump_size) {
+    const void* stack_dump, size_t stack_dump_size) {
```

Comment:
Can we please stick to `uint64_t`, like everywhere else in this module?

Reply:
done.